README.md

```diff
@@ -257,8 +246,8 @@ namespace SlicerTester.Console
 }
 ```
 
-### Dictionary&lt;string, dynamic> Index(Dictionary&lt;string, dynamic> index, bool autoCreateFields = false)
-Index data to existing entities or create new entities, if necessary. This method corresponds to a [POST request at /index](http://panel.slicingdice.com/docs/#api-details-api-endpoints-post-index).
+### Dictionary&lt;string, dynamic> Insert(Dictionary&lt;string, dynamic> data, bool autoCreateColumns = false)
```

Comment:
"autoCreateColumns" parameter should be removed.
Check this PR: (https://github.com/SlicingDice/slicingdice-dot-net/pull/12)